backend/src/layer/data/swka_parser/swka_resolver.rs

```diff
@@ -0,0 +1,88 @@
+use crate::interface::mensa_parser::ParseError;
+use futures::future::join_all;
+use reqwest::Client;
+use std::time::Duration;
+use tracing::log::debug;
+
+pub struct SwKaResolver;
+
+impl SwKaResolver {
+    pub const fn new() -> Self {
+        Self
+    }
+
+    fn get_client() -> Result<Client, ParseError> {
+        let client = Client::builder() // dont change or tests will fail!
+            .danger_accept_invalid_certs(true) //probably not needed
+            .timeout(Duration::from_millis(1000))
+            .connect_timeout(Duration::from_millis(1000))
```

Comment:
this is afik redudnant as it is included in the `timeout`
https://docs.rs/reqwest/latest/reqwest/struct.ClientBuilder.html#method.timeout

Reply:
timeout at least 4 sec